velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -780,14 +780,10 @@ uint32_t HiveDataSink::appendWriter(const HiveWriterId& id) {
         insertTableHandle_->serdeParameters().end());
   }
 
-  options->processConfigs(*hiveConfig_->config(), *connectorSessionProperties);
-
-  const auto& sessionTimeZoneName = connectorQueryCtx_->sessionTimezone();
-  if (!sessionTimeZoneName.empty()) {
-    options->sessionTimezone = tz::locateZone(sessionTimeZoneName);
```

Comment:
@rui-mo Why do we not need this default anymore? Is it always set in the connectorQueryCtx?

Reply:
`options->sessionTimezone` was removed following this suggestion: https://github.com/facebookincubator/velox/pull/11949#discussion_r1896983263.